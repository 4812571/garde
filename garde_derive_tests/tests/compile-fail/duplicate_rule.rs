#[derive(garde::Validate)]
struct Test<'a> {
    #[garde(ascii, ascii)]
    field: &'a str,
}

fn main() {}
