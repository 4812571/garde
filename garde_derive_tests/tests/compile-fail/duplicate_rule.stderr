error: duplicate rule `ascii`
 --> tests/compile-fail/duplicate_rule.rs:3:20
  |
3 |     #[garde(ascii, ascii)]
  |                    ^^^^^
